docs/using/examples/example.py

```diff
@@ -0,0 +1,60 @@
+import httplib
+import os
+import ssl
+import tempfile
+
+# NOTE - to run this example, you need at least python 2.7.9
+
+# Define the control IP, port, and the certificates for authentication.
+
+CONTROL_SERVICE = os.environ.get(
```

Comment:
If we are going to go with manual testing (see other comments, and consider the results of our past attempts at doing manual testing for our docs / features) then it would be easier if there were a manual script to follow.

For example if we had a `flocker-provision` type tool which created a cluster which could be used with the default environment variables. Maybe hack `run-acceptance-tests` to help (I think it'll do most of the heavy lifting).